vdr/api/v1/api.go

```diff
@@ -88,10 +91,21 @@ func (a *Wrapper) AddNewVerificationMethod(ctx context.Context, request AddNewVe
 		opts = &VerificationMethodRelationship{}
 	}
 
-	vm, err := a.VDR.NutsDocumentManager().AddVerificationMethod(ctx, *d, opts.ToFlags(didnuts.DefaultKeyFlags()))
+	vms, err := a.SubjectManager.AddVerificationMethod(ctx, d.String(), opts.ToFlags(didnuts.DefaultKeyFlags()))
 	if err != nil {
 		return nil, err
 	}
+	var vm *did.VerificationMethod
+	for _, m := range vms {
+		if m.ID.DID.String() == request.Did {
+			vm = &m
+			break
+		}
+	}
+	if vm == nil {
+		return nil, resolver.ErrNotFound
```

Comment:
this would rather be a really weird internal error instead of not found? (you did something on a DID that didn't exist, but no error was returned).

Reply:
`"verification method added for subject: %s but not for DID: %s, do not use the V1 API for non-nuts DIDs"`